vnext/JSI/Desktop/ChakraJsiRuntime_core.cpp

```diff
@@ -219,65 +139,51 @@ void ChakraRuntime::setupNativePromiseContinuation() noexcept {
   }
 }
 
-#if !defined(CHAKRACORE_UWP)
-// This is very wierd. This should match with the definition of VS_VERSIONINFO
-// as defined in
-// https://docs.microsoft.com/en-us/windows/desktop/menurc/vs-versioninfo I
-// can't find a way to include the actual definition of VS_VERSIONINFO
-// TODO :: Re-evaluate this strategy.
-struct FileVersionInfoResource {
-  uint16_t len;
-  uint16_t valLen;
-  uint16_t type;
-  wchar_t key[_countof(L"VS_VERSION_INFO")];
-  uint16_t padding1;
-  VS_FIXEDFILEINFO fixedFileInfo;
-  uint32_t padding2;
-};
-#endif
-
-// TODO :: This code is mostly copied from the old ChakraExecutor flow, and not
-// verified for reliability yet.
-// TODO :: Re-evaluate this strategy of finding the dll version for versioning
-// the runtime.
-/*static*/ void ChakraRuntime::initRuntimeVersion() noexcept {
-  // This code is win32 only at the moment. We will need to change this
-  // line if we want to support UWP.
-#if !defined(CHAKRACORE_UWP)
-  constexpr wchar_t chakraDllName[] = L"ChakraCore.dll";
+void ChakraRuntime::startDebuggingIfNeeded() {
+  auto &args = runtimeArgs();
+  if (args.enableDebugging) {
+    auto port =
```

Comment:
>port [](start = 9, length = 4)

(nit) it would be nice to standardize naming. Here we have variations:
port
args.debuggerPort
m_debugPort
DebuggerDefaultPort

Perhaps it should be:
debuggerPort
args.debuggerPort
m_debuggerPort
DefaultDebuggerPort

Reply:
Similarly, it would be nice to standardize naming for "debuggerRuntimeName" concept

---
In reply to: [333111079](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333111079) [](ancestors = 333111079)

Reply:
The code for supporting in-proc debugging needs a lot of clean up, but I think that should be done in a separate PR. I'll keep these feedback in mind when I get to it.

---
In reply to: [333111822](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333111822) [](ancestors = 333111822,333111079)